src/Data/Integer/Show.agda

```diff
@@ -0,0 +1,24 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Showing natural numbers
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Integer.Show where
+
+open import Data.Integer.Base using (ℤ; +_; -[1+_])
+open import Data.Nat.Base using (suc)
+open import Data.Nat.Show using () renaming (show to showℕ)
+open import Data.String.Base using (String; _++_)
+
+------------------------------------------------------------------------
+-- Show
+
+-- Decimal notation: conversion back and forth with characters
+-- Time complexity is O(log₁₀(n))
+
+show : ℤ → String
```

Comment:
Err, it turns out we already have it in `Data.Integer`

https://github.com/agda/agda-stdlib/blob/e4a715f757766294ab1d417e946ee1f7a9560903/src/Data/Integer.agda#L28

I think we should deprecate that one and keep this one. For the reasons outlined above, we don't really want all the string dependencies cluttering `Data.Integer`